x-pack/plugins/security_solution/public/detections/components/take_action_dropdown/index.tsx

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState, useCallback, useMemo } from 'react';
+import { EuiContextMenu, EuiButton, EuiPopover } from '@elastic/eui';
+import type { ExceptionListType } from '@kbn/securitysolution-io-ts-list-types';
+
+import { find } from 'lodash/fp';
+import { TAKE_ACTION } from '../alerts_table/alerts_utility_bar/translations';
+
+import { TimelineEventsDetailsItem, TimelineNonEcsData } from '../../../../common';
+import { useExceptionActions } from '../alerts_table/timeline_actions/use_add_exception_actions';
+import { useAlertsActions } from '../alerts_table/timeline_actions/use_alerts_actions';
+import { useInvestigateInTimeline } from '../alerts_table/timeline_actions/use_investigate_in_timeline';
+/* Todo: Uncomment case action after getAddToCaseAction is split into action and modal
```

Comment:
This is for another PR?

Reply:
That was planed to be part of this PR, but there is still another dependency.